src/wazuh_modules/agent_upgrade/manager/wm_agent_upgrade_upgrades.c

```diff
@@ -141,12 +141,8 @@ void wm_agent_upgrade_prepare_upgrades() {
 
         node = wm_agent_upgrade_get_next_node(&index, node);
 
-        if (!queue_push_ex(upgrade_queue, agent_task)) {
-            wm_agent_upgrade_remove_entry(agent_key, 0);
-        } else {
-            mterror(WM_AGENT_UPGRADE_LOGTAG, WM_UPGRADE_UPGRADE_QUEUE_FULL, agent_key);
```

Comment:
You can also remove `WM_UPGRADE_UPGRADE_QUEUE_FULL` from the `error_message.h` file.